test/unit/css.js

```diff
@@ -641,7 +641,7 @@ QUnit.test( "show/hide detached nodes", function( assert ) {
 	span.remove();
 } );
 
-QUnit[ document.body.attachShadow ? "test" : "skip" ]( "show/hide shadow child nodes", function( assert ) {
+QUnit[ document.body.getRootNode ? "test" : "skip" ]( "show/hide shadow child nodes", function( assert ) {
```

Comment:
Just checking for `attachShadow` here would run the test in iOS 10.0 where it'd fail. We either need to check both or blacklist specific browsers by user agent (as it's just tests).

Reply:
Thanks for the pointer, though! I didn't notice these tests would fail in Firefox 60 ESR.

Reply:
Whoops, got it now after inspecting further... `getNodeRoot` is not used here, but will get invoked within the library code when swapping the `display` style :+1: 

So... yeah, this may turns tricky for Firefox.